build.sbt

```diff
@@ -134,7 +133,8 @@ lazy val client = (project in file("target/clients/java"))
     openApiAdditionalProperties := Map(
       "library" -> "native",
       "useJakartaEe" -> "true",
-      "hideGenerationTimestamp" -> "true"),
+      "hideGenerationTimestamp" -> "true",
+      "openApiNullable" -> "false"),
```

Comment:
By default, the generator uses [this open api nullable library](https://github.com/OpenAPITools/jackson-databind-nullable) which I think is what needs jackson 2.17.1, by disabling this, we can make do with 2.15.0